lib/lock_manager.rb

```diff
@@ -9,8 +9,8 @@ def initialize(options = {})
     @options = options
   end
 
-  def lock(host, user, reason = nil)
-    LockManager::Worker.new(connection, host).lock(user, reason)
+  def lock(host, user, reason = nil, ttl = nil)
```

Comment:
Can you have multiple optional variables in ruby without keyword args or an options hash?